core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -259,6 +259,19 @@ private static Manifest loadLinkedManifest(File archive) throws IOException {
         return new PluginWrapper(pluginManager, archive, manifest, baseResourceURL,
                 createClassLoader(paths, dependencyLoader, atts), disableFile, dependencies, optionalDependencies);
     }
+    
+    /**
+     * Returns all the bundled plugin dependencies for a particular Jenkins version
+     */
+    public static List<PluginWrapper.Dependency> getPreviouslyBundledDependencies(String jenkinsVersion) {
```

Comment:
:ant: the naming is misleading. Whether these were _bundled_ in 1.x or not is irrelevant—only whether they were _detached_ from core. (There are some bundled plugins which were never detached, and we could have unbundled detached plugins though as a matter of policy I think we never did.)